x/feerefunder/keeper/keeper.go

```diff
@@ -204,6 +227,23 @@ func (k Keeper) StoreFeeInfo(ctx sdk.Context, feeInfo types.FeeInfo) {
 	store.Set(types.GetFeePacketKey(feeInfo.PacketId), bzFeeInfo)
 }
 
+func (k Keeper) GetPayerInfo(ctx sdk.Context, sender string, payer string) (types.PayerInfo, error) {
+	senderAddr, err := sdk.AccAddressFromBech32(sender)
+	if err != nil {
+		k.Logger(ctx).Debug("Transfer: failed to parse sender address", "sender", sender)
+		return types.PayerInfo{}, sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "failed to parse address: %s", sender)
+	}
+	feePayerAddr, err := sdk.AccAddressFromBech32(payer)
+	if payer != "" && err != nil {
```

Comment:
why try to create an address from the payer if the payer is empty? this leads to confusing possible empty but not nil `FeePayer` in the `types.PayerInfo` below which is not good: in go, it's usually a prepared ready-to-use value given or nil.
please consider having the `FeePayer` equal to nil in case payer is not provided here. this will also improve [this part of code](https://github.com/neutron-org/neutron/blob/62a86f1b1943bce8140080fe512cf9a6454b0a02/x/feerefunder/keeper/keeper.go#L72): will be no need to check if its empty, only if its nil.

Reply:
good point!